docs/guides/built-ins/actions/approve/example.yml

```
@@ -0,0 +1,8 @@
+api-version: reviewpad.com/v3.x
+
+workflows:
+  - name: trivial-changes
+    if:
+      - $hasFileName("README.md")
+    then:
+      - $approve("This pull request only contains trivial changes")
```

Sounds like a good example to use `LTGM` :)

```suggestion
      - $approve("LGTM")
```